api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,62 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
```

Comment:
Hmm, `reads` and `writes` sounds wrong, even though it complies to the letter of the guideline. The entries there are not reads/writes.

Maybe `readers` and `writers` (together with `admins`), with the meaning that each item in that list indicates who is allowed to read/write/administrate?